src/components/Connection.tsx

```diff
@@ -651,6 +675,9 @@ const Connection: React.FC<ConnectionProps> = ({
       toast.error("Failed to save data. Please try again.");
     }
   };
+  
+  
+  
 
```

Comment:
Why do we have so many blank lines here?

Reply:
thankyou for noticing @lorforlinux, I have fixed issues.